repos.d/gentoo/guru.yaml

```diff
@@ -0,0 +1,37 @@
+###########################################################################
+# GURU
+###########################################################################
+- name: guru
+  sortname: gentoo~guru  # place right after Gentoo
+  type: repository
+  desc: GURU
+  statsgroup: Gentoo
+  family: gentoo
+  ruleset: [ gentoo, guru ]
+  color: '62548f'
+  update_period: 12h
+  minpackages: 100
+  default_maintainer: guru@gentoo.org
+  sources:
+    - name: guru
+      fetcher: GitFetcher
+      parser: GentooGitParser
+      url: https://github.com/gentoo-mirror/guru.git
+      require_md5cache_metadata: true
+      require_xml_metadata: false
+      sparse_checkout: [ '**/*.ebuild', '**/metadata.xml', 'metadata/md5-cache/*' ]
+  repolinks:
+    - desc: GURU Ebuild repository entirely maintained by Gentoo users
+      url: https://wiki.gentoo.org/wiki/Project:GURU
+    - desc: GURU repository of new packages maintained collaboratively by users
+      url: https://gitweb.gentoo.org/repo/proj/guru.git/
+    - desc: [MIRROR] GURU repository of new packages maintained collaboratively by users
+      url: https://github.com/gentoo/guru
+    - decs: Article "GURU: a new model of contributing to Gentoo (WIP)"
+      url: https://dev.gentoo.org/~mgorny/articles/guru-a-new-model-of-contributing-to-gentoo.html
+  packagelinks:
+    - desc: Package details
+      url: 'https://gpo.zugaina.org/Overlays/guru/{srcname}'
+    - desc: View ebuild
+      url: 'https://gitweb.gentoo.org/repo/proj/guru.git/tree/{srcname}/{srcname|basename}-{rawversion}.ebuild'
+  tags: [ all ]
```

Comment:
production tag is needed

Reply:
add `production` and `gentoo-overlays` tags